src/media/UVideo.pas

```diff
@@ -701,6 +701,9 @@ function TVideo_FFmpeg.Open(const FileName : IPath): boolean;
     // fail if called concurrently by different threads.
     FFmpegCore.LockAVCodec();
     try
+        // by setting this explicitly to 0, it won't default to a single thread
+        fCodecContext^.thread_count := 0;
```

Comment:
this at the very least needs a `{$IF LIBAVCODEC_VERSION >= ...` wrapped around it, unless you also edit the other avcodec header fies, otherwise this won't compile.